modules/common/configs/proto/vehicle_config.proto

```diff
@@ -65,9 +65,9 @@ message VehicleParam {
   optional float max_abs_speed_when_stopped = 18;
 
   // minimum value get from chassis.brake, in percentage
-  optional float brake_deadzone = 19;
+  optional double brake_deadzone = 19;
   // minimum value get from chassis.throttle, in percentage
-  optional float throttle_deadzone = 20;
+  optional double throttle_deadzone = 20;
```

Comment:
Is this proto ever published and recorded in data? If yes, this change will break compatibility.

Reply:
No, it hasn't. This is a non-recorded proto and the throttle/brake_deadzone are newly added to it.